cypress/integration/Combobox.spec.ts

```diff
@@ -37,7 +37,7 @@ describe('Combobox', () => {
     });
 
     it('should not show menu', () => {
-      cy.findByRole('listbox').should('not.be.visible');
+      cy.findByRole('listbox').should('not.exist');
```

Comment:
All these were due to a breaking change in [Cypress v6](https://docs.cypress.io/guides/references/changelog#6-0-0).

This "fix" did find legitimate issues with our tests, but I feel like we've lost some semantics. I want to know if the element is no longer accessible to the user. Whether it is still in the DOM or just hidden is an implementation detail. Perhaps I'll make an `be.inaccessible` assertion that does this.